### ISSUE TITLE
node merging heuristics, for <mn> in the base, like 123^5

### ISSUE THREAD

**Issue by hbghlyj** [ronkok/Temml#40]
Temml separates "123" into two <mn> nodes
```
  <mrow>
    <mn>12</mn>
    <msup>
      <mn>3</mn>
      <mn>5</mn>
    </msup>
  </mrow>
```
[MathJax](https://www.mathjax.org/#demo) and [LaTeXML](https://latexml.mathweb.org/editor) and [Pandoc](https://pandoc.org/try/) treat "123" as a whole base, which can simplify MathML code
```
  <msup>
    <mn>123</mn>
    <mn>5</mn>
  </msup>
```

**Comment by ronkok** [comment]
Hmm... At one point, Temml parsed this expression as you suggest. Then came issue #13. My fix for that issue has created this issue.

I'll look into it. I'm sure it's fixable, but a few days may go by before I get to it.

**Comment by ronkok** [comment]
@hbghlyj, Your issues are getting more obscure. That's encouraging.

This issue is resolved by release v0.10.19.

**Comment by hbghlyj** [comment]
Number with a decimal point, for example `12.3` is converted to `<mn>12.3</mn>`
but `12.3^4` is converted to `<mn>12</mn><mi>.</mi><msup><mn>3</mn><mn>4</mn></msup>`
in [MathJax](https://www.mathjax.org/#demo) `12.3^4` is converted to `<msup><mn>12.3</mn><mn>4</mn></msup>`
